translations/CONTRIBUTING.hindi.md

```diff
@@ -0,0 +1,151 @@
+
+<a name="areas"> </a>
+
+## 4. योगदान के क्षेत्र
+
+बेशक, हम हसुरा के सभी घटकों में योगदान की सराहना करते हैं। हालांकि, हमने तीन क्षेत्रों की पहचान की है जो योगदान के लिए विशेष रूप से उपयुक्त हैं।
+
+### दस्तावेज़
+
+हमारा लक्ष्य हमारे डॉक्स को विस्तृत और अद्यतन रखना है। यदि आप ऐसा करने में हमारी मदद करना चाहते हैं, तो हम किसी भी तरह के योगदान के लिए आभारी है। 
+
+- लापता जानकारी की रिपोर्ट करें
+
+- मौजूदा दस्तावेज़ में गलतियों को ठीक करें
+
+- दस्तावेज़ को बढ़ाने में हमारी मदद करें
+
+डॉक्स के लिए योगदानकर्ता गाइड [docs/CONTRIBUTING.md](docs/CONTRIBUTING.md) पर पाया जा सकता है।
+
+### सामुदायिक जानकारी
+
+जबसे हमने अपना [सीखना पृष्ठ](https://hasura.io/learn/) लॉन्च किया, हम योगदानों से ख़ुशि हैं:
+
+- मौजूदा ट्यूटोरियल में गलतियों को ठीक करें
+
+- नए ट्यूटोरियल जोड़ें (कृपया हमें पहले आपके विचार बताएं ताकि डुप्लीकेट ना हो)
+
+जानने की रिपॉजिटरी का README [यहां](https://github.com/hasura/learn-graphql) पाया जा सकता है।
+
+सीखने की जानकारी के अलावा, तकनीकी सामुदायिक जानकारी में योगदान करने के तीन अन्य तरीके हैं:
+
+- [बॉयलरप्लेट्स](https://github.com/hasura/graphql-engine/tree/master/community/boilerplates)
+
+- [नमूना ऐप्स](https://github.com/hasura/graphql-engine/tree/master/community/sample-apps)
+
+- [उपकरण](community/tools)
+
+यदि आप समुदाय को योगदान देना चाहते हैं निम्नलिखित माध्यमों से, तो हमारी [समुदाय विकि](https://github.com/hasura/graphql-engine/wiki/Community-Wiki) देखें।:
+
+- टेक्निकल ब्लॉगपोस्ट लिखना
+
+- किसी इवेंट में बोलना
+
+- एक कार्यशाला का आयोजन
+
+यदि आप ऊपर बताई गई चीज़ की अलावा कुछ और जोड़ना चाहते हैं, एक पुल रिक्वेस्ट ज़रूर भेजें।
+
+### हसुरा सीएलआई
+
+हमारे पास सीएलआई पर कुछ मुद्दे हैं जो ओपन सोर्स के योगदान के लिए उपयुक्त हैं। यदि आप गो जानते हैं या आप इस पर काम करके सीखना चाहते हैं, तो इन [मुद्दों](https://github.com/hasura/graphql-engine/issues?q=is%3Aopen+is%3Aissue+label%3Ac%2Fcli+label%3A%22help+wanted%22) को देखें।
+
+CLI रिपॉजिटरी की README को [यहाँ](https://github.com/hasura/graphql-engine/tree/master/cli) पाया जा सकता है।
+
+<a name="ways"> </a>
+
+## 5. योगदान के तरीके
+
+### किसी समस्या की रिपोर्ट करना
+
+- सुनिश्चित करें कि आपने नवीनतम संस्करण के खिलाफ परीक्षण किया हैं। यह संभव है कि हम आपके द्वारा अनुभव किए जा रहे बग को पहले ही सही कर लिए हैं।
+
+- समस्या को पुन: उत्पन्न करने के लिए कदम प्रदान करें, साथ ही पोस्टग्रेज संस्करण, ग्राफ्क्यूएल-इंजन संस्करण और प्रदाता(हरोकू, डॉकर, आदि) जिसे आप चला रहे हैं।
+
+- यदि आवश्यक हो तो कृपया सर्वर के लॉग भी शामिल करें।
+
+### किसी मुद्दे पर काम करना
+
+- हम [फोर्क-और-ब्रांच गिट वर्कफ़्लो](https://blog.scottlowe.org/2015/01/27/use-fork-branch-git-workflow/) का उपयोग करते हैं।
+
+- कृपया सुनिश्चित करें कि आप जो काम कर रहे हैं, उससे जुड़ा एक इश्यू है।
+
+- यदि आप किसी मुद्दे पर काम कर रहे हैं, तो कृपया टिप्पणी करें ताकि दूसरों के द्वारा डुप्लिकेट काम ना हो।
+
+- अपने कमिट को स्क्वाश करें और अंत में कमिट मैसेज में `फिक्स #<इश्यू-नो>` या `क्लोज़ #<इश्यू-नो>` का इस्तेमाल करें।
+
+    उदाहरण: `हर चीज़ का उत्तर हल करें (फिक्स #42)` या `हर चीज़ का उत्तर हल करें, फिक्स #42`
```

Comment:
As mentioned above, Please change the above to:
* `resolve answers to everything (fix #42)`
* `resolve answers to everything, fix #42`